lua/neorg/modules/core/summary/module.lua

```diff
@@ -89,7 +89,8 @@ module.load = function()
                     if not norgname then
                         norgname = filename
                     end
-                    norgname = norgname:gsub("^" .. ws_root, "")
+                    local escaped_ws_root = ws_root:gsub("([%(%)%.%%%+%-%*%?%[%^%$%]])", "%%%1")
+                    norgname = norgname:gsub("^" .. escaped_ws_root, "")
```

Comment:
Had same problem, and this fixed it for me with no regex xD
```suggestion
                    norgname = string.sub(norgname, string.len(ws_root) + 1)
```

Reply:
yeah this seems much nicer, thanks! :white_flower: